amazon_msk/pyproject.toml

```diff
@@ -35,13 +35,12 @@ dependencies = [
 dynamic = [
     "version",
 ]
-
```

Comment:
@ofek do you know what this is about? did we change the tool we use to write toml files?

Reply:
Somebody must have manually added an extra line at some point